opencti-platform/opencti-graphql/src/database/generate-message.js

```diff
@@ -56,12 +58,26 @@ export const generateDeleteMessage = (instance) => {
   return generateCreateDeleteMessage(EVENT_TYPE_DELETE, instance);
 };
 
-export const generateUpdateMessage = (entityType, inputs) => {
+export const generateUpdateMessage = async (context, entityType, inputs) => {
   const inputsByOperations = R.groupBy((m) => m.operation ?? UPDATE_OPERATION_REPLACE, inputs);
   const patchElements = Object.entries(inputsByOperations);
   if (patchElements.length === 0) {
     throw UnsupportedError('[OPENCTI] Error generating update message with empty inputs');
   }
+
+  const authorizedMembersIds = patchElements.slice(0, 3).flatMap(([,operations]) => {
```

Comment:
this whole block looks a bit too complicated for just history logging, do we really need this ?